docker/README.md

```diff
@@ -68,41 +68,38 @@ docker rm -f mms
 
 ## Available pre-built continers
```

Comment:
This whole section talks about a lot of containers in future tense, as in 'will be built'. I believe these containers are already available so we should update language to reflect that correctly.

Reply:
Updated this.